cmd/glasskube/cmd/uninstall.go

```diff
@@ -78,11 +79,26 @@ var uninstallCmd = &cobra.Command{
 			}
 			fmt.Fprintln(os.Stderr, "Uninstallation started in background")
 		} else {
-			if err := uninstaller.UninstallBlocking(ctx, pkg, uninstallCmdOptions.DryRun); err != nil {
-				fmt.Fprintf(os.Stderr, "\n❌ An error occurred during uninstallation:\n\n%v\n", err)
-				cliutils.ExitWithError()
+			// delete the namespace only if it is namespace scoped and there are no other packages in the namespace since
+			// cluster packages delete the namespace once they are deleted
+			deleteNamespace := uninstallCmdOptions.DeleteNamespace && pkg.IsNamespaceScoped()
```

Comment:
If DeleteNamespace is set and the package is not namespace-scoped, the command shoul exit with an error